extension/js/popup.js

```diff
@@ -1,16 +1,61 @@
-function popup() {
-  document.querySelector("#generate-alias").addEventListener("click", async (e) => {
-    const newRelayAddressResponse = await browser.runtime.sendMessage({
-      method: "makeRelayAddress",
-    });
+/* global browser */
+
+function showSignUpPanel() {
+  const signUpOrInPanel = document.querySelector(".sign-up-panel");
+  return signUpOrInPanel.classList.remove("hidden");
+}
+
+function showRelayPanel() {
+  const relayPanel = document.querySelector(".signed-in-panel");
+  return relayPanel.classList.remove("hidden");
+}
+
+async function popup() {
+  const userApiToken = await browser.storage.local.get("apiToken");
+  if (!userApiToken) {
```

Comment:
Based on my struggle with `browser.storage.local.get` in the other parts of the add-on, I don't think checking for `!userApiToken` is enough here. I believe `userApiToken` will be an empty object, which will always evaluate to true. You may need to do something more like `!userApiToken.hasOwnProperty('userApiToken')`?

Reply:
Yeah, this `userApiToken` check always evaluations to true. Even in a brand new `web-ext run` profile with nothing in the extension storage, I get the relay panel instead of the sign-up panel.

<img width="1392" alt="image" src="https://user-images.githubusercontent.com/71928/80213425-c4e8cd00-85fe-11ea-9260-278ae7bed5bf.png">